src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+
+const calculate = (obj, btnName) => {
+  if (btnName === '+/-') {
+    (obj.total * (-1)).toString();
+    (obj.next * (-1)).toString();
+    return obj;
+  }
+
+  if (obj.operation) {
+    return {
+      total: operate(obj.total, obj.next, obj.operation),
+      next: null,
+      operation: btnName,
+    };
+  }
+  return obj;
+};
+
+export default calculate;
```

Comment:
- [ ] This module does not contain enough conditions to satisfy all operations from `operate.js`. Kindly fix this by adding conditions that produce a particular result when a specific button and obj are set as arguments for this module calculate function.